### ISSUE TITLE
PoloniexApi Live OrderBook changes

### ISSUE THREAD

**Issue by misthema** [Jojatekok/PoloniexApi.Net#4]
Would be happy to have this feature! I really need real-time order book and trade history changes. Live tickers wont help with that... :) I tried implementing it myself but got really confused 'cuz it works little different than the TickerChanges.

**Comment by roeynissim** [comment]
were you able to subscribe to the feed?

**Comment by misthema** [comment]
Yes. I actually got it eorking, both ordervook and trade history. I can paste rhe code here later (too lazy to create repo for it...)

**Comment by roeynissim** [comment]
I also did it, but i'm getting orderbook and trade messages from yesterday, am i doing something wrong?

**Comment by misthema** [comment]
Make sure you are converting the timestamps correctly. Poloniex uses UTC.

**Comment by roeynissim** [comment]
its not a timezone issue, when I compare prices I get from the feed to prices from the site I get different numbers
